appengine/standard/migration/incoming/README.md

```diff
@@ -0,0 +1,16 @@
+## App Engine to App Engine Request Sample
+
+This sample application shows how an App Engine for Python 2.7 app can verify
+the caller's identity for a request from an App Engine for Python 2.7 or
+App Engine for Python 3.7 app.
+
+Requests from an App Engine for Python 2.7 app that uses `urlfetch` have
```

Comment:
Replace 'uses' with 'use':  

Request from an App Engine for Python 2.7 app that use

Reply:
Fixed. I was thinking "app that uses" instead of "requests that use".